system/Filters/FilterInterface.php

```diff
@@ -31,7 +31,7 @@ interface FilterInterface
      *
      * @param array|null $arguments
      *
-     * @return mixed
+     * @return RequestInterface|ResponseInterface|string|void
```

Comment:
This is a bit odd, but correct because now we can return any non-empty.

```suggestion
     * @return RequestInterface|ResponseInterface|mixed|void
```

Reply:
I'm okay with this change, but I would not mind if we left it as it was (only `mixed`).
I don't have a strong opinion.